docs/developer-guide/syntax-reference.rst

```diff
@@ -795,6 +800,11 @@ for deletion and if the topic format is AVRO, delete the corresponding Avro sche
 deleted, too. Topic deletion is asynchronous, and actual removal from brokers
```

Comment:
While you're. Could you fix this line and the above? `delete the corresponding Avro schema is deleted, too`. I think the first delete word should not be there.